jupyterlab_pullrequests/base.py

```diff
@@ -1,8 +1,15 @@
 from typing import List, NamedTuple, Optional
 
+import entrypoints
 from traitlets import Enum, Unicode, default
 from traitlets.config import Configurable
 
+# Supported third-party services
+MANAGERS = {}
+
+for entry in entrypoints.get_group_all("pull_requests_manager"):
```

Comment:
Nitpick, but the [specification](https://packaging.python.org/specifications/entry-points/#data-model) suggests:

```
new group should use names starting with a PyPI name owned by the consumer 
project, followed by .. Group names must be one or more groups of letters, 
numbers and underscores, separated by dots (regex ^\w+(\.\w+)*$).
```

Further, as we might be changing stuff in the future, having an API version baked in would probably be good as the "last line of defense" against big API changes, so perhaps `jupyterlab_pullrequests.manager_v1`.

Reply:
Thanks for pointing me to the specification. I updated the name with your suggestion.